content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+{{< warning >}}
+This guide relies heavily on the
+[stable plugin ca certificate](/docs/tasks/security/cert-management/plugin-ca-cert/) feature. Istio
+does not offer an industry standard recommendation for certificate management. Please consult your
+information security team when using any plugin certificate.  The determination of your company
+policies for certificate management is the choice of your company.
+{{< /warning >}}
+
```

Comment:
```suggestion
{{< tip >}}
This workflow is tested and validated. The Istio authors feel this workflow is suitable for experimentation but not production. Like all alpha features, this workflow is subject to change.
This is a useful tip from an expert
{{< /tip >}}

```

Reply:
I think moving the banner text to the top of the page (so it only shows on this page) is a better idea, and then remove the banner.

Reply:
thanks @ericvn I have addressed the comment.

Cheers,
-steve